firefox-ios/Client/Assets/RemoteSettingsData/RemoteSettingsFetchConfig.json

```diff
@@ -1,11 +1,19 @@
 {
-  "rules": [
+  "collections": [
```

Comment:
don't forget to update [RemoteSettingsFetchConfig.swift](https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Application/RemoteSettings/RemoteSettingsFetchConfig.swift) as you changed rules to collections

Reply:
Aha Thanks. I did in fact forget that 🙈

Reply:
Done in [7670e5f](https://github.com/mozilla-mobile/firefox-ios/pull/23199/commits/7670e5f9aabd407362a4f2457bad21f540f01981)